docs/sources/panels-visualizations/visualizations/trend/index.md

```diff
@@ -0,0 +1,31 @@
+---
+keywords:
+  - grafana
+  - graph panel
+  - trend panel
+  - documentation
+  - guide
+  - graph
+title: Trend
+weight: 90
```

Comment:
Is there any reason that this page is not in alphabetical order or can we move it? (I realize there are few pages in this section that are out of alpha order, but this one is easy to fix since it'll be the last one)

Reply:
I'm ok with moving it to be in alphabetic order